packages/ui/email-pcd-ui/src/CardBody.tsx

```diff
@@ -2,7 +2,7 @@ import { EmailPCD, getEmailAddress } from "@pcd/email-pcd";
 import { styled } from "@pcd/passport-ui";
 import { PCDUI } from "@pcd/pcd-types";
 
-export const EmailPCDUI: PCDUI = {
+export const EmailPCDUI: PCDUI<EmailPCD, unknown> = {
```

Comment:
Not sure if this is already the case but you can make the second type parameter be default `unknown` which would make specifying like you've done here optional. Another thing is that the default type for the 2nd parameter might be better defined as `never` rather than `unknown`